rust/arrow/src/compute/kernels/comparison.rs

```diff
@@ -570,27 +570,26 @@ where
         ));
     }
 
-    let num_bytes = bit_util::ceil(left_len, 8);
+    let num_bytes = utils::ceil(left_len, 8);
 
     let not_both_null_bit_buffer =
         match combine_option_bitmap(left.data_ref(), right.data_ref(), left_len)? {
             Some(buff) => buff,
             None => new_all_set_buffer(num_bytes),
         };
-    let not_both_null_bitmap = not_both_null_bit_buffer.data();
+    let _not_both_null_bitmap = not_both_null_bit_buffer.data();
```

Comment:
I wonder if there is some reason to keep this statement? It seems unused

Reply:
Things that I forgot along the lines.

Reply:
Addressed too.